docs/docs/APIdocs/layers/nn/Loss Functions/ClassNLLCriterion.md

```diff
@@ -0,0 +1,93 @@
+## ClassNLLCriterion
+
+**Scala:**
+```scala
+val criterion = ClassNLLCriterion()
+```
+**Python:**
+```python
+criterion = ClassNLLCriterion()
+```
+
+The negative log likelihood criterion. It is useful to train a classification problem with n
+classes. If provided, the optional argument weights should be a 1D Tensor assigning weight to
+each of the classes. This is particularly useful when you have an unbalanced training set.
+
+The input given through a forward() is expected to contain log-probabilities of each class:
+input has to be a 1D Tensor of size n. Obtaining log-probabilities in a neural network is easily
+achieved by adding a LogSoftMax layer in the last layer of your neural network. You may use
+CrossEntropyCriterion instead, if you prefer not to add an extra layer to your network. This
```

Comment:
Shall we quote (\` \`) functions, parameters, class names, etc.? e.g. `forward`, `n`, and `backward` and `LogSoftMax`, `CrossEntropyCriterion`